cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -42,7 +42,7 @@ angular.module(PKG.name + '.commons')
           'ng-trim': 'false',
           'ng-model': 'model',
           'rows': '{{myconfig["widget-attributes"].rows}}',
-          placeholder: '{{::myconfig["widget-attributes"].default}}'
+          placeholder: '{{::myconfig["widget-attributes"].placeholder}}'
```

Comment:
Talked to @elfenheart. Right now we use `default` to have a default value and when the default value is set we no longer show the placeholder.